rpcs3/CMakeLists.txt

```diff
@@ -89,8 +90,7 @@ target_link_libraries(rpcs3 PRIVATE ${ADDITIONAL_LIBS})
 
 # Unix display manager
 if(X11_FOUND)
-    target_include_directories(rpcs3 PUBLIC ${X11_INCLUDE_DIR})
-    target_link_libraries(rpcs3 PRIVATE ${X11_LIBRARIES})
+    target_link_libraries(rpcs3 PRIVATE X11::X11)
```

Comment:
What does this have to do with mingw?

Reply:
Nothing. Do you want a separate PR?

Reply:
Since this is unrelated, it would make more sense for regression testing

Reply:
Reverted.